aws/dms/tasks.yml

```
@@ -34,6 +34,7 @@ cron:
 - dashboard.contained_levels
 - dashboard.level_sources_multi_types
 - dashboard.paired_user_levels
+- dashboard.parental_permission_requests
```

I think this should cover it. By removing the unneded `parent_email` this table is safe to load into the `dashboard` non-pii schema.

```suggestion
- dashboard.parental_permission_requests:
    remove_column: [parent_email]
```